src/cr/cube/crunch_cube.py

```diff
@@ -1198,3 +1189,9 @@ def pvals(self, weighted=True, prune=False, hs_dims=None):
     def scale_means(self):
         '''Get cube means.'''
         return ScaleMeans(self).data
+
+    def _get_slices(self):
+        if self.ndim < 3:
+            return [CubeSlice(self, 0)]
+
+        return [CubeSlice(self, i) for i, _ in enumerate(self.as_array())]
```

Comment:
`return [CubeSlice(self, i) for i in range(len(self.as_array()))]`

Is there an easier way to get `len(self.as_array)`?  Or does CA potentially make it very complicated?

Reply:
Yes, we can iterate over labels, which is faster...